src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/TvpTest.cs

```diff
@@ -41,7 +41,12 @@ public class TvpTest
         // data value and server consts
         private string _connStr;
 
+        /// <summary>
+        /// ToDo: This test is marked as windows only for now, cause of different short date pattern behaviour in linux as two digits for year.
+        /// Probably solution is adding a separate base line for linux.
+        /// </summary>
         [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
I realize the difficulty of figuring out how to reconcile the dates across Windows and Linux, but we just recently got this test working again, cross-plat (for the same underlying reason). So I hate to see it disabled again, even if it's only being disabled on Linux.

I'm not excited about the idea of adding a separate baseline for Linux. But maybe you can intercept the exception message somewhere in the test stack and replace the few different 2-digit-year dates with equivalent 4-digit versions.

Reply:
Nobody likes this type of limitations. But, the method of comparison has two main steps, first generating a complete output file and on the next level, the messages are compared with our baselines, line by line without any sense about their contents. In my opinion, we need to redesign this part to have access to the contents for overloading in a special scenario like this.

Reply:
I'm very familiar with this test. I'm not suggesting we post-process the entire output. The message in the output that has changed all flows through DateTimeVariantTest.DisplayError() when the test runs. There is already a bit of message massaging in that method to ensure messages are consistent across targets. I'm suggesting we can add very specific checks for the new bulk copy error message there and adjust it before it is written to the output file. Look at the last commit to this file (and related ones) and you can see the effort we made to make this work across platforms.

Reply:
Can we move this particular test out of BSL, out of this class and also handle Unix formats so we wouldn't have to disable the entire thing and we can also test this across platforms?